manifest.json

```
@@ -3,9 +3,10 @@
 	"id": "mopidy",
 	"packaging_format": 1,
 	"description": {
-		"en": "An extensible music server to play audio from your music collection and various streaming services"
+		"en": "Mopidy - is an extensible music server written in Python.",
+		"fr": "Mopidy - est un serveur de musique extensible écrit en Python"
 	},
-	"version": "3.1.1-ynh1",
+	"version": "1.0.2",
```

```suggestion
	"version": "3.1.1~ynh2",
```

looking at https://github.com/siwinter/mopidy_ynh/blob/58aa835463fb012282d8aad1709c11ef46f2a11b/scripts/_common.sh#L37, it seems that version is 3.1.1